### ISSUE TITLE
This file cannot get the time code.

### ISSUE THREAD

**Issue by jsls2517** [x42/libltc#72]
[output_right_channel.wav.zip](https://github.com/user-attachments/files/16929148/output_right_channel.wav.zip)

**Comment by x42** [comment]
That looks like a super noisy signal which also is very quiet (-47.64dBFS).

I managed to decode it after adding 40dB gain and a 2nd order Low Pass filter at 5kHz (with Q=0.9) and then feeding the result into the decoder.  It contains timecode from 19:48:48:00 to 19:49:17:22

How did you record that sound?

**Comment by jsls2517** [comment]
This should be generated by the hardware timer, but I can't confirm the model now. I will try it according to your method.

**Comment by x42** [comment]
Could it be that you directly connected BNC from a hardware generator to a soundcard's low impedance input?

There definitely are issues, both signal level and impedance, and perhaps also a missing termination. It is far from a clean signal.

**Comment by jsls2517** [comment]
It is no longer possible to trace how the file was generated, but I can parse it using other tools.

**Comment by jsls2517** [comment]
> It is no longer possible to trace how the file was generated, but I can parse it using other tools.

<img width="1151" alt="image" src="https://github.com/user-attachments/assets/3366e859-9486-438c-8a19-d57eb24ee891">

**Comment by jsls2517** [comment]
> Could it be that you directly connected BNC from a hardware generator to a soundcard's low impedance input?
> 
> There definitely are issues, both signal level and impedance, and perhaps also a missing termination. It is far from a clean signal.

How exactly is it realized? Can you teach me? After using ffmpeg to process such audio, can it be obtained accurately?

**Comment by x42** [comment]
The issue already happened when recording the signal. You need to fix the hardware. Get a proper cable [1], and then set the input level so that the AD receives perhaps -10dBFS peak.

As for the current file, you can try with ffmpeg or sox or whatever you favorite tool is to add significant gain and low pass filter the signal. This is outside of the scope of the decoder library and not usually required for a properly recorded LTC signal.


[1] e.g. https://ambient.de/en/collections/timecode-cables

**Comment by jsls2517** [comment]
Yes, the hardware problem should be solved. But the old files must be obtained somehow. I tried sox but couldn't get the correct timecode.

`sox ~/Downloads/input_right_channel.wav ~/Downloads/output_right_channel.wav gain 40 lowpass 5000 0.9q`

Is it the wrong way I use it?
